hedera/tests/test_api.py

```diff
@@ -18,7 +18,8 @@
 class PersonalVocabularyQuickAddAPITest(APITestCase):
 
     def setUp(self):
-        self.created_user = User.objects.create_user(username=f"test_user{randrange(100)}", email=f"test_user{randrange(100)}@test.com", password="password")
```

Comment:
I experienced test failures using `randrange(100)`, as sometimes generated `usernames` would actually clash, so switching to UUIDs makes things more random.